### PR TITLE
Remove OE logging

### PR DESCRIPTION
**Pull request by eddyashton**
Swallow the OE logging so it doesn't log "logging is not enabled" in the wrong format.

Goodbye forever to this message:

```
2021-04-16T16:39:20+0100.498499Z [(H)WARN] tid(0x7ff6502ee600) | In-enclave logging is not supported. To enable, please add 

from "openenclave/edl/logging.edl" import *;

in the edl file.
 [/source/openenclave/host/sgx/create.c:oe_create_enclave:1178]
```

### CODE REVIEW

**Review by wintersteiger — APPROVED** [review]
❤️

File: src/host/enclave.h [hunk 1]
```diff
@@ -56,6 +70,8 @@ namespace host
       }
       else
       {
+        oe_log_set_callback(nullptr, nop_oe_logger);
```
Comment (achamayou) [inline src/host/enclave.h:73]: Can we keep oe logging when VERBOSE_LOGGING is on?

**Review by achamayou — APPROVED** [review]

File: src/host/enclave.h (hunk 1 above)
Comment (letmaik) [reply src/host/enclave.h:73]: What's the resolution here?

Comment (eddyashton) [reply src/host/enclave.h:73]: I missed this - will disable this nop logger when `VERBOSE_LOGGING` is on so we restore the old behaviour. I believe that's still a single line though, unless apps can turn on more OE logging themselves?

**Pull request merged**